spec/type_aliases/ensure_package_spec.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'Stdlib::Ensure::Package' do
+  describe 'valid handling' do
+    [
+      'present',
+      'absent',
+      'purged',
+      'disabled',
+      'installed',
+      'latest',
+      '1',
+      '1.1',
```

Comment:
Could you also add a version range, which is supported by at least some package providers (at least DNF):
```suggestion
      '>=6.0',
```

It'll be more for documentation purposes/regression testing than anything else.